src/pages/my-courses/MyCourses.tsx

```diff
@@ -128,19 +130,17 @@ const MyCourses = () => {
 
   const { items: coursesItems, count: coursesCount } = coursesResponse
 
-  if (coursesLoading) {
-    return <Loader pageLoad />
-  }
-
   return (
     <PageWrapper>
       <Typography sx={styles.title}>{t('myCoursesPage.title')}</Typography>
       <Box sx={styles.divider}></Box>
-      <AddCourseWithInput />
+      <AddCourseWithInput fetchData={fetchData} searchRef={searchTitle} />
       {!coursesItems.length && !coursesLoading ? (
         <NotFoundResults
           description={t('myCoursesPage.notFound.description')}
         />
+      ) : coursesLoading ? (
```

Comment:
Maybe better to move this condition higher or to put this into (brackets).
You have "1 new issue" there.
<img width="528" alt="Screenshot 2023-12-15 at 12 22 29" src="https://github.com/ita-social-projects/SpaceToStudy-Client/assets/104988390/20a82064-ffaf-415c-8934-b66c8bd5289d">